kubernetes/iterated-training/shuffle-and-export.sh

```diff
@@ -12,6 +12,5 @@ while true; do
 
   ITERATION_DIR=/"$VOLUME_NAME"/victimplay/"$RUN_NAME"/iteration-"$ITERATION"
   run_until_curriculum_done "$ITERATION_DIR" \
-    /go_attack/kubernetes/shuffle-and-export.sh "$RUN_NAME"-"$ITERATION" \
-    "$RUN_NAME"/iteration-"$ITERATION" "$VOLUME_NAME"
+    /go_attack/kubernetes/shuffle-and-export.sh "$RUN_NAME"/iteration-"$ITERATION" "$VOLUME_NAME"
```

Comment:
IIUC, `shuffle-and-export.sh` used to take a separate run name and directory argument. Now it takes just one (along with the volume name), and we're setting the run name to be what the directory used to be. But since the `RUN_NAME` is only used as part of directory arguments, it's fine if the run name has a `/` in it.